libs/utils/utils.js

```diff
@@ -706,22 +712,40 @@ async function checkForPageMods() {
       .filter((path) => path?.trim());
   }
 
-  let martechLoaded = false;
-  if (targetEnabled) {
-    martechLoaded = await loadMartech({ persEnabled: true, persManifests, targetMd });
+  const utils = {
+    createTag,
+    getConfig,
+    loadScript,
+    loadLink,
+    updateConfig,
+    loadStyle,
+    getMetadata,
+  };
+
+  const { mep: mepOverride } = Object.fromEntries(PAGE_URL.searchParams);
+  const previewPage = window.location.host.includes('.hlx.page') || window.location.host.includes('localhost');
```

Comment:
I would lean on the config object for this info, no need to calculate this again.

Reply:
Hey @auniverseaway , this update stemmed from a discussion with @chrischrischris to remove code and calculations from the critical path unless needed.  So 725 and 726 are deciding if further code should be imported or not.  But if you prefer, I can move that function back into utils

Reply:
```suggestion
  const { env } = getConfig();
  const previewPage = env.name === 'stage' || env.name === 'local';
```